runtime/jvmti/jvmtiGeneral.c

```diff
@@ -186,7 +186,7 @@ jvmtiGetVersionNumber(jvmtiEnv* env,
 done:
 	if (NULL != version_ptr) {
 #if JAVA_SPEC_VERSION >= 15
-		*version_ptr = JVMTI_VERSION_15;
+		*version_ptr = JVMTI_VERSION;
```

Comment:
Should this just always use JVMTI_VERSION now?

Reply:
I think `JVMTI_1_2_3_SPEC_VERSION` is still required by JDK8, `JVMTI_VERSION_11` can be folded though.

Reply:
PR updated.